05_Intro_to_Source_Detection.ipynb

```diff
@@ -89,10 +78,36 @@
     "import lsst.afw.image as afwImage\n",
     "import lsst.afw.display as afwDisplay\n",
     "import lsst.afw.table as afwTable\n",
-    "import lsst.geom as afwGeom\n",
+    "import lsst.geom as geom\n",
     "\n",
+    "# Import Pipeline tasks \n",
+    "from lsst.pipe.tasks.characterizeImage import CharacterizeImageTask\n",
+    "from lsst.pipe.tasks.calibrate import CalibrateTask\n",
+    "from lsst.meas.algorithms.detection import SourceDetectionTask\n",
+    "from lsst.meas.deblender import SourceDeblendTask\n",
+    "from lsst.meas.base import SingleFrameMeasurementTask"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
     "# Use lsst.afw.display with the matplotlib backend\n",
-    "afwDisplay.setDefaultBackend('matplotlib')"
+    "afwDisplay.setDefaultBackend('matplotlib')\n",
+    "plt.rcParams['figure.figsize'] = (8.0, 8.0)"
```

Comment:
I think it makes more sense to set the rcParams once here rather than in the various cells below. I set the default figure size to (8, 8), since I think this is what we use in other notebooks.

Reply:
That does make sense, agree.